websites/I/India Today News/presence.ts

```diff
@@ -0,0 +1,541 @@
+const presence = new Presence({
+		clientId: "1051110545723506718",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			play: "general.playing",
+			pause: "general.paused",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+
+interface iFrameData {
+	duration: number;
+	currentTime: number;
+	paused: boolean;
+}
+
+let video: iFrameData = {
+	duration: 0,
+	currentTime: 0,
+	paused: true,
+};
+
+presence.on(
+	"iFrameData",
+	(data: { duration: number; currentTime: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+enum Assets {
+	Logo = "https://i.imgur.com/JJHnTn3.png",
+	Reading = "https://i.imgur.com/8vMPNni.png",
+	Searching = "https://i.imgur.com/oGQtnIY.png",
+	Pause = "https://i.imgur.com/0A75vqT.png",
+	Play = "https://i.imgur.com/Dj5dekr.png",
+	Live = "https://i.imgur.com/wtweZhK.png",
+}
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			details: "Unsupported page",
+			smallImageKey: Assets.Logo,
+			smallImageText: "India Today News",
+		},
+		strings: Awaited<ReturnType<typeof getStrings>>,
+		oldLang: string = null;
+
+	const { href, pathname } = document.location,
+		[showTimestamp, showButtons, showAuthor, newLang] = await Promise.all([
+			presence.getSetting<boolean>("timestamp"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("author"),
+			presence.getSetting<string>("lang").catch(() => "en"),
+		]),
+		pages: Record<string, PresenceData> = {
+			"/elections": {
+				details: "Viewing election news",
+				buttons: [
+					{
+						label: "View election news",
+						url: href,
+					},
+				],
+			},
+			"/india": {
```

Comment:
Also look into replacing some or most of the records by using this (if possbile)
`document.querySelector('[class="jsx-559305e56e2f7896 active"]')?.textContent`
![Screenshot_16](https://user-images.githubusercontent.com/42322979/208242613-45ebc9d1-65a4-4b73-9268-621726739dae.png)
![Screenshot_17](https://user-images.githubusercontent.com/42322979/208242615-d275933b-e295-48a1-ba2c-cbdf4d539ee5.png)